IngresseSDK/Extensions/Encodable+Dictionary.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Encodable+Dictionary.swift
+//  IngresseSDK
+//
+//  Created by Fernando Ferreira on 23/04/20.
```

Comment:
Geralmente a gente deixa assim no cabeçalho:

```suggestion
//
//  Copyright © 2020 ingresse. All rights reserved.
//
```